week-1/Homework/mandatory/2-logic-error.js

```diff
@@ -1,16 +1,16 @@
 // The syntax for this function is valid but it has an error, find it and fix it.
 
+/* trim is a function so have to use dot to make it work */
```

Comment:
Well done. Just remember when a function is part of an object, it's called a 'method' but your fix is still correct.